src/Validation.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Omnipay\PayZen;
+
+use DateTime;
+
+class Validation
+{
+    /**
+     * @param string $date
+     * @param string $format
+     * @return bool
+     */
+    public static function validateDate($date, $format = 'Y-m-d H:i:s')
+    {
+        $d = DateTime::createFromFormat($format, $date);
+        return $d && $d->format($format) == $date;
+    }
+
+    /**
+     * @param string $date
+     * @param string $start
+     * @param string $end
+     * @return bool
+     */
+    public static function isDateInPeriod($date, $start, $end)
```

Comment:
maybe we should validate that all dates are a real date.
if all value are wrong (date = 'aaa', start = 'bbb', end = 'ccc') we will return true because all value will be equals to false and indeed false is equal to false :D